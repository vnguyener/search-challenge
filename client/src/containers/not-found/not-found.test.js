import React from 'react';
import { Provider } from 'react-redux';
import { configure } from 'enzyme';
import renderer from 'react-test-renderer';
import configureMockStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import Adapter from 'enzyme-adapter-react-16';
import NotFoundPage from '.';

configure({ adapter: new Adapter() });
const mockStore = configureMockStore([thunk]);

describe('not found container', () => {
  it('renders correctly', () => {
    const store = mockStore();

    const wrapper = renderer
      .create(
        <Provider store={store}>
          <NotFoundPage />
        </Provider>
      )
      .toJSON();

    expect(wrapper).toMatchSnapshot();
  });
});
