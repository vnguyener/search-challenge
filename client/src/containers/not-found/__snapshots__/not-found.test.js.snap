// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`not found container renders correctly 1`] = `
<main
  className="not-found-container"
>
  <h2>
    404
  </h2>
  <p>
    Oops, looks like you're lost.
  </p>
  <button
    className="minimal-button"
    onClick={[Function]}
    style={
      Object {
        "border": "1px solid",
        "borderRadius": "8px",
        "padding": "12px",
        "textTransform": "uppercase",
      }
    }
  >
    Back to Results
  </button>
</main>
`;
